packages/projects-docs/pages/learn/environment/vm.mdx

```diff
@@ -38,25 +36,36 @@ Note that, the first time you import a project to CodeSandbox, it can take longe
 
 ## Persistence
 
-Everything you save in `/project` is guaranteed to be persisted between reboots, hibernations and forks. Every file change outside of `/project` will usually be persisted, but there is a chance that those changes will be cleared.
+We use [Dev Containers](https://codesandbox.io/docs/learn/environment/devcontainers) by default for all repositories and Devboxes. Your project is under `/workspaces/workspace`, and your home folder is under `/root`. Under the hood, the paths are in `/project`.
+
+<Callout>The workspace folder is not always `/workspaces/workspace`; it is dependent on the Dev Container configuration.</Callout>
+
+Everything in `/workspaces/workspace` is guaranteed to be persisted between reboots, hibernations and forks. We persist the workspace folder and the Docker engine state, like images, volumes, etc.
+
+Every file change outside of `/workspaces/workspace` will usually be persisted, but there is a chance that those changes will be cleared.
```

Comment:
This is the other way round. Outside the workspace the changes are not usually persisted. It might be persisted between restarts but there is no guarantee. So maybe we can remove this line.